build/images/scripts/start_ovs_netdev

```diff
@@ -53,7 +60,8 @@ function add_br_phy {
     ip addr flush dev eth0 2>/dev/null
     ip link set eth0 up
     ip route add default via "$gw" dev br-phy
-    iptables -t raw -A PREROUTING -i eth0 -j DROP
```

Comment:
I'll wait for your comments for the rest of the PR, but did you confirm that there is no need for a similar rule with the new bridge? With the previous configuration, this rule would prevent the original packet from going through the regular TCP / IP stack (while a copy is processed by OVS userspace) and messing up conntrack. You have to check manually as it doesn't always create issues in CI.